applicationconfig/client/src/main/java/com/azure/applicationconfig/ConfigurationAsyncClient.java

```diff
@@ -290,6 +290,15 @@ public static ConfigurationAsyncClientBuilder builder() {
         return result.flatMapMany(this::extractAndFetchConfigurationSettings);
     }
 
+    static String getSelectQuery(SettingFields[] set) {
```

Comment:
I noticed this was made package-private. Did we end up using this in another class?

Reply:
Leftovers from the subclass prototype, changed to private.